TranchedLockupPresale/migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,57 @@
+const TranchedLockupPresale = artifacts.require('./TranchedLockupPresale.sol');
+const CustomToken = artifacts.require('./CustomToken.sol');
+
+let currentTime = Math.floor(+new Date()/1000); // UNIX current timestamp in seconds
+module.exports = function(deployer, network, accounts) {
+    const openingTime = currentTime; // epoch timestamp of deployment

```

Comment:
we should at some extra minutes, because the deployment procedure will take some time (and therefore reverts when it tries to deploy the contract)

Reply:
So:
6: const openingTime = currentTime + 2 // extra time to facilitate time for the deployment procedure
?

Reply:
2 second won't be enough (truffle initializing can take minutes on Ropsten). I suggest + 2*60 (= 2 minutes)